rosidl_adapter/rosidl_adapter/parser.py

```diff
@@ -448,36 +448,39 @@ def parse_message_file(pkg_name, interface_filename):
             pkg_name, msg_name, h.read())
 
 
+def extract_file_level_comments(message_string):
+    lines = message_string.splitlines()
+    index = next(
+        (i for i, v in enumerate(lines) if not v.startswith(COMMENT_DELIMITER)), -1)
+    if index != -1:
+        file_level_comments = lines[0:index]
+        other = lines[index:]
```

Comment:
@ivanpauno nit^N: consider `file_content` instead of `other`.

Reply:
Done in https://github.com/ros2/rosidl/pull/632/commits/32ef5a327b8054d0b5de88d0ab25b2de417af797.